source/sync/data-model/create-a-schema.txt

```diff
@@ -142,6 +142,10 @@ Create an App Services Schema from a Realm Object Model
 
             :ref:`Define a Realm Object Schema - Realm Kotlin SDK <kotlin-supported-types>`
 
+         .. tab::
```

Comment:
Bah. Unfortunately, adding C++ to the tab group is now causing it to be too wide to display all of the tabs. It has pushed React Native off the side of the tab list, and the tab list doesn't appear to scroll. (At least not for me in Chrome.)

We should probably change this from a tab list to a bullet list, instead, given this limitation. I'll ask about it in Slack to see if we have any work planned to support longer tab lists so we can go back to tabs for this in the future.